legal-api/src/legal_api/reports/report.py

```diff
@@ -240,6 +240,8 @@ def _get_template_filename(self):
         return '{}.html'.format(file_name)
 
     def _get_template_data(self):
+        from legal_api.services import flags  # pylint: disable=import-outside-toplevel
```

Comment:
is there any reason for importing it here?

Reply:
I followed how the import was done in the business.py file that I was referencing but I can make it a top-level import.

Reply:
yes. please move to the top